File: C# Examples/13 Document Manager API/SWDMReferences/SolidWorksMacro.cs
using SolidWorks.Interop.sldworks;

using SWDMProperties; //namespace from shared MyDocMan.cs

namespace SWDMReferences
{
    public partial class SolidWorksMacro
    {
        public void Main()
        {
            MyDocMan MyDM = new MyDocMan();

            //browse for a file
            string filePath = MyDM.BrowseForDoc();
            MyDM.GetRefs(filePath, true);
            return;
        }

        // The SldWorks swApp variable is pre-assigned for you.
        public SldWorks swApp;

    }
}

File: 13 Document Manager API/SWDMReferences/SolidWorksMacro.vb
Imports SolidWorks.Interop.sldworks

Partial Class SolidWorksMacro

    Public Sub main()
      Dim MyDM As New MyDocMan

      'Browse for a file
      Dim FilePath As String
      FilePath = MyDM.BrowseForDoc()

      MyDM.GetRefs(FilePath, True)

    End Sub

    ''' <summary>
    ''' The SldWorks swApp variable is pre-assigned for you.
    ''' </summary>
    Public swApp As SldWorks


End Class
